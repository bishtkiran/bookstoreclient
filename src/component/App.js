import axios from 'axios';
import '../App.css';
import React from 'react';
import Layout from './layout/Layout';
import BookContainer from './book/BookContainer';
import baseUrl from '../config';


  const App = () => {
    axios.get(`${baseUrl}/api/v1/books`).then(books => {
      console.log("Books", books);
    })
      return (
        <Layout>
         <BookContainer />
        </Layout>
      );
    }


export default App;
