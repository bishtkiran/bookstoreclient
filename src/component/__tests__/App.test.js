import { render, screen } from '@testing-library/react';
import App from '../App';
import renderWithRedux from '../../util/testUtil';
import React from "react";

test('Application', () => {
    const asFragment = renderWithRedux(<App />, {});
    expect(asFragment).toMatchSnapshot();
});
