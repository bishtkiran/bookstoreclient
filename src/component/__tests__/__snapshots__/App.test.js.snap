// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Application 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="MuiBox-root MuiBox-root-1"
      >
        <div
          class="MuiBox-root MuiBox-root-2"
        >
          <header
            class="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary mui-fixed MuiPaper-elevation4"
          >
            <div
              class="MuiToolbar-root MuiToolbar-dense MuiToolbar-gutters"
            >
              <h6
                class="MuiTypography-root MuiTypography-h6 MuiTypography-colorInherit"
              >
                Book Store
              </h6>
            </div>
          </header>
        </div>
        <div
          class="MuiBox-root MuiBox-root-3"
        >
          <div
            class="MuiBox-root MuiBox-root-10 makeStyles-bookContainer-4"
          >
            <div
              class="MuiBox-root MuiBox-root-11 makeStyles-bookFilter-5"
            >
              <div
                class="MuiPaper-root makeStyles-bookFilterPaper-6 MuiPaper-elevation1 MuiPaper-rounded"
              >
                <p
                  class="MuiTypography-root MuiTypography-body1"
                >
                  Search Book Filters
                </p>
                <div
                  class="MuiBox-root MuiBox-root-12"
                >
                  <div
                    class="MuiFormControl-root MuiTextField-root"
                    data-testid="book-title-input"
                  >
                    <label
                      class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
                      data-shrink="false"
                      for="book-search"
                      id="book-search-label"
                    >
                      Enter book title
                    </label>
                    <div
                      class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
                    >
                      <input
                        aria-invalid="false"
                        class="MuiInputBase-input MuiOutlinedInput-input"
                        id="book-search"
                        placeholder="Enter book title"
                        type="text"
                        value=""
                      />
                      <fieldset
                        aria-hidden="true"
                        class="PrivateNotchedOutline-root-13 MuiOutlinedInput-notchedOutline"
                      >
                        <legend
                          class="PrivateNotchedOutline-legendLabelled-15"
                        >
                          <span>
                            Enter book title
                          </span>
                        </legend>
                      </fieldset>
                    </div>
                  </div>
                </div>
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiButton-label"
                  >
                    Search
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </div>
            </div>
            <div
              class="MuiBox-root MuiBox-root-17 makeStyles-bookList-7"
            >
              <div
                class="MuiBox-root MuiBox-root-19"
              >
                <span
                  class="MuiSkeleton-root MuiSkeleton-rect MuiSkeleton-pulse"
                  data-testid="book-loader"
                  style="width: 80%; height: 200px;"
                />
              </div>
              <div
                class="MuiBox-root MuiBox-root-18 makeStyles-bookList-7"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="MuiBox-root MuiBox-root-1"
    >
      <div
        class="MuiBox-root MuiBox-root-2"
      >
        <header
          class="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary mui-fixed MuiPaper-elevation4"
        >
          <div
            class="MuiToolbar-root MuiToolbar-dense MuiToolbar-gutters"
          >
            <h6
              class="MuiTypography-root MuiTypography-h6 MuiTypography-colorInherit"
            >
              Book Store
            </h6>
          </div>
        </header>
      </div>
      <div
        class="MuiBox-root MuiBox-root-3"
      >
        <div
          class="MuiBox-root MuiBox-root-10 makeStyles-bookContainer-4"
        >
          <div
            class="MuiBox-root MuiBox-root-11 makeStyles-bookFilter-5"
          >
            <div
              class="MuiPaper-root makeStyles-bookFilterPaper-6 MuiPaper-elevation1 MuiPaper-rounded"
            >
              <p
                class="MuiTypography-root MuiTypography-body1"
              >
                Search Book Filters
              </p>
              <div
                class="MuiBox-root MuiBox-root-12"
              >
                <div
                  class="MuiFormControl-root MuiTextField-root"
                  data-testid="book-title-input"
                >
                  <label
                    class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
                    data-shrink="false"
                    for="book-search"
                    id="book-search-label"
                  >
                    Enter book title
                  </label>
                  <div
                    class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
                  >
                    <input
                      aria-invalid="false"
                      class="MuiInputBase-input MuiOutlinedInput-input"
                      id="book-search"
                      placeholder="Enter book title"
                      type="text"
                      value=""
                    />
                    <fieldset
                      aria-hidden="true"
                      class="PrivateNotchedOutline-root-13 MuiOutlinedInput-notchedOutline"
                    >
                      <legend
                        class="PrivateNotchedOutline-legendLabelled-15"
                      >
                        <span>
                          Enter book title
                        </span>
                      </legend>
                    </fieldset>
                  </div>
                </div>
              </div>
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiButton-label"
                >
                  Search
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
            </div>
          </div>
          <div
            class="MuiBox-root MuiBox-root-17 makeStyles-bookList-7"
          >
            <div
              class="MuiBox-root MuiBox-root-19"
            >
              <span
                class="MuiSkeleton-root MuiSkeleton-rect MuiSkeleton-pulse"
                data-testid="book-loader"
                style="width: 80%; height: 200px;"
              />
            </div>
            <div
              class="MuiBox-root MuiBox-root-18 makeStyles-bookList-7"
            />
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
